wpilibc/src/main/native/cpp/GenericHID.cpp

```diff
@@ -83,3 +84,7 @@ void GenericHID::SetRumble(RumbleType type, double value) {
   }
   HAL_SetJoystickOutputs(m_port, m_outputs, m_leftRumble, m_rightRumble);
 }
+
+frc2::JoystickButton GenericHID::GetButton(int button) {
```

Comment:
I really don't like GenericHID depending on commands, as it breaks encapsulation and tightly couples the classes.  What's the reason for doing it this way?